client/gutenberg/extensions/presets/jetpack/index.json

```diff
@@ -2,5 +2,6 @@
   "markdown",
   "publicize",
   "simple-payments",
+  "subscriptions",
```

Comment:
Considering that this block is in development, we might want to move this to `index-beta` instead.